subsys/bluetooth/Kconfig

```diff
@@ -178,4 +178,13 @@ config BT_COMPANY_ID
 
 source "subsys/bluetooth/mesh/Kconfig"
 
+config BT_DISABLE
+	bool "Enable Bluetooth disable functionality"
+	select DYNAMIC_DIRECT_INTERRUPTS if CONFIG_BT_LLL_VENDOR_NORDIC
+	help
+	  Enable the Bluetooth disable API which would stop all Bluetooth
+	  activity and reset the Bluetooth controller. If using the internal
+	  controller the interrupts will be released and the application can
+	  connect them using dynamic interrupts.
```

Comment:
Do we need to mention here, the overhead on the BLE IRQ latency due to the dynamic interrupts @joerchan ?
I.e. the fact that there's a sequence of: __get_IPSR(), calculate vector-address, jump to vector address.